File: src/FrontEnd/NgClient/src/app/shared/models/account.ts
export interface IAccount {
    id: string;
    userName: string;
    lastName: string;
    firstName: string;
    middleName: string;
    role: string;
    organizationId: string;
    phoneNumber: string;
    officePhone: string;
    email: string;
    specializationId: string;
    isActive: boolean;
}

export interface IRegisterAccount {
    userName: string;
    password: string;
    confirmPassword: string;
    lastName: string;
    firstName: string;
    middleName: string;
    role: string;
    organizationId: string;
    phoneNumber: string;
    officePhone: string;
    email: string;
    specializationId: string;
}

export interface IRole {
    id: string;
    name: string;
}

File: src/IdentityServer/CORE/IdentityServer.Domain/Account.cs
using Microsoft.AspNetCore.Identity;

namespace IdentityServer.Domain
{
    public class Account : IdentityUser
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public IdentityRole Role { get; set; }
        public string PasswordSalt { get; set; }
        public string RoleId { get; set; }
        public bool IsActive { get; set; } = true;
        public string OrganizationId { get; set; }
        public string SpecializationId { get; set; }
        public string OfficePhone { get; set; }
    }
}
